### PR TITLE
Popup Window PR

### PR DESCRIPTION
**Pull request by ntimanu**
In this project we:

- Add desktop popup window.
- Mobile popup window.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,7 +102,7 @@ <h2>Tonic</h2>
             />
           </div>
           <div class="card1txt">
-            <h2>Tonic</h2>
+            <h2>Multi-Post Stories</h2>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: [REQUIRED] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. You should loop through your projectContent array and for each item kindly make sure you generate the card that will be append to this projects section.

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
Hi Team @ntimanu  and @HussenYesufAlli   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ntimanu) [reply index.html]: Done

**Review by Munsa1 — APPROVED** [review]
Hi @ntimanu, 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Nice UI Design :heavy_check_mark: 
Dynamic Content As Requested :heavy_check_mark: 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**